src/cron/schedule-code-review.js

```diff
@@ -26,15 +24,14 @@ const getMappedStatusText = (key) => {
 };
 
 /**
- * Checks threads for a specific status and sends reminders in all text channels.
- *
- * @param {Client} client - The Discord.js client instance.
- * @param {string} statusText - The status to look for in thread titles.
+ * Checks threads with a specific status and sends reminders if necessary.
+ * @param {Client} client - Instance of the Discord.js client.
+ * @param {string} statusKey - Key of the status to check.
+ * @returns {string|null}
  */
-const checkThreadsForReview = async (client, statusText) => {
+const checkThreadsForStatus = async (client, statusKey) => {
```

Comment:
following with https://github.com/TeamNovaSoft/novabot/pull/113/files#r1958137240, you should accept the config parameter here
```suggestion
const checkThreadsForStatus = async (client, statusKey, statusConfig) => {
```